pkg/controller/util_test.go

```diff
@@ -255,6 +257,186 @@ var _ = Describe("setConditionFromPod", func() {
+	})
+
+	It("Should return global preallocation setting if not defined in DV or SC", func() {
+		client := createClient(createCDIConfigWithGlobalPreallocation(true))
+		dv := createDataVolumeWithStorageClass("test-dv", "test-ns", "test-class")
+		preallocation := GetPreallocation(client, dv)
+		Expect(preallocation).To(BeTrue())
+
+		client = createClient(createCDIConfigWithGlobalPreallocation(false))
+		preallocation = GetPreallocation(client, dv)
+		Expect(preallocation).To(BeFalse())
+	})
+
+	It("Should be false when niether DV nor Config defines preallocation", func() {
+		client := createClient(createCDIConfig("test"))
+		dv := createDataVolumeWithStorageClass("test-dv", "test-ns", "test-class")
+		preallocation := GetPreallocation(client, dv)
+		Expect(preallocation).To(BeFalse())
+	})
+})
+
+var _ = Describe("GetDefaultStorageClass", func() {
+	It("Should return the default storage class name", func() {
+		client := createClient(
+			createStorageClass("test-storage-class-1", nil),
+			createStorageClass("test-storage-class-2", map[string]string{
+				AnnDefaultStorageClass: "true",
+			}),
+		)
+		sc, _ := GetDefaultStorageClass(client)
+		Expect(sc.Name).To(Equal("test-storage-class-2"))
+	})
+
+	It("Should return nil if there's not default storage class", func() {
+		client := createClient(
+			createStorageClass("test-storage-class-1", nil),
+			createStorageClass("test-storage-class-2", nil),
+		)
+		sc, _ := GetDefaultStorageClass(client)
+		Expect(sc).To(BeNil())
+	})
+})
+
+var _ = Describe("GetStorageClassNameForDV", func() {
+	It("Should return DV's StorageClass if defined", func() {
+		client := createClient()
+		dv := createDataVolumeWithStorageClass("test-name", "test-ns", "test-storage-class")
+		scName := GetStorageClassNameForDV(client, dv)
+		Expect(scName).To(Equal("test-storage-class"))
+	})
+
+	It("Should return default StorageClass if not specified in DV or VM", func() {
+		client := createClient(
+			createStorageClass("test-storage-class-1", nil),
+			createStorageClass("test-storage-class-2", map[string]string{
+				AnnDefaultStorageClass: "true",
+			}),
+		)
+		dv := createDataVolume("test-name", "test-ns")
+		scName := GetStorageClassNameForDV(client, dv)
+		Expect(scName).To(Equal("test-storage-class-2"))
+	})
+
+	It("Should return empty StorageClass if not defined anywhere", func() {
+		client := createClient(
+			createStorageClass("test-storage-class-1", nil),
+			createStorageClass("test-storage-class-2", nil),
+		)
+		dv := createDataVolume("test-name", "test-ns")
+		scName := GetStorageClassNameForDV(client, dv)
+		Expect(scName).To(Equal(""))
+	})
+})
+
+func addOwnerToDV(dv *cdiv1.DataVolume, ownerName string) {
+	dv.ObjectMeta.OwnerReferences = []metav1.OwnerReference{
+		{
+			APIVersion: "v1",
+			Kind:       "VirtualMachine",
+			Name:       ownerName,
+		},
+	}
+}
+
+func createVirtualMachineWithStorageClassName(name, ns, dvName, storageClassName string) *kubevirtv1.VirtualMachine {
```

Comment:
This doesn't seem to be used anywhere, and if it was, I would be giving some push back on it. Anything from kubevirt/kubevirt, is going to be a problem in cdi, our main user is kubevirt, but we are not dependent on them

Reply:
Oh, yes, this was used by the part of code that was removed. Removing this as well.